Makefile

```diff
@@ -282,6 +282,16 @@ else
 	npx react-native run-ios
 endif
 
+show-ios-devices: export TARGET := ios
+show-ios-devices: ##shows connected ios devices and their UDID
```

Comment:
The comment format is:
```
{target_name}: ##@{section_name} {comment_text}
```
Because we want to make this work:
https://github.com/status-im/status-mobile/blob/6c1929096c1ced0a857c9e021b2b5dcac70a0842/Makefile#L16-L27

I'd put `show-ios-devices` in `other` and `run-ios-device` in `run`.